qcdb/driver/endorsed_plugins.py

```diff
@@ -17,6 +17,8 @@
     for mtd in psi4.driver.proc_table.procedures['energy']:
         procedures['energy']['psi4'][mtd.lower()] = intf_psi4.run_psi4
         procedures['energy']['psi4']['p4-' + mtd.lower()] = intf_psi4.run_psi4
+    procedures['energy']['psi4']['p4-mrccsd(t)'] = intf_psi4.run_psi4
+    procedures['energy']['psi4']['p4-mrccsdtq'] = intf_psi4.run_psi4
```

Comment:
This is psi4's fault for handling in `parse_arbitrary_order` rather than entering the methods into the `procedures` dict. No nondisruptive sol'n, so fine for now.